clients/gtest/trsm_gtest.yaml

```diff
@@ -60,6 +60,15 @@ Definitions:
   - &large_memory_matrix_size_range
     - { M: 8320, N: 128, lda: 8320, ldb: 8320 }
 
+  - &size_t_left_matrix_size_range
+#   - { M: 4, N: 46435, lda: 4, ldb: 46435 }
+    - { M: 46345, N: 4, lda: 46345, ldb: 46345 }
+#   - { M: 47000, N: 4, lda: 47000, ldb: 47000 } # calls rocblas_internal_gemm_template with batch_count=367, stride_a=6016128
+
+  - &size_t_right_matrix_size_range
+    - { M: 4, N: 46345, lda: 46345, ldb: 4 }
```

Comment:
These tests timed out for me on gfx90a, but passed when I increased the timeout threshold. I'm testing all other configs as well just to be safe.

Reply:
@daineAMD  Naveen's speedup for the initialization of the triangle matrix in https://github.com/ROCmSoftwarePlatform/rocBLAS-internal/pull/1551 . This PR is in the develop branch but it is not in release/rocm-rel-5.5. Tests will run faster on the develop branch than on release/rocm-rel-5.5.